### PR TITLE
improvement(contextualmenu): update contextual menu source of truth

### PR DESCRIPTION
**Pull request by EdisonKleeen**
Update contextual menu source of truth to handle cross linking better

### CODE REVIEW

File: libs/atomic-elements/src/lib/components/GridSection/GridSection.tsx [hunk 1]
```diff
@@ -209,7 +210,7 @@ function TableHeader(props: TableHeaderProps) {
                     const { value } = e.target;
                     handleChange(attr.name, value);
                   }}
-                  label={attr.name}
+                  label={attr.label || attr.name}
```
Comment (jorgekleeen) [inline libs/atomic-elements/src/lib/components/GridSection/GridSection.tsx:213]: What is the difference between `label` and `name`?

Comment (EdisonKleeen) [reply libs/atomic-elements/src/lib/components/GridSection/GridSection.tsx:213]: you can have a label created by selecting an aggregation (Sum of expenses) and it is optional

Comment (EdisonKleeen) [reply libs/atomic-elements/src/lib/components/GridSection/GridSection.tsx:213]: If you don't have a label, you'll show the name as usual

Comment (jorgekleeen) [reply libs/atomic-elements/src/lib/components/GridSection/GridSection.tsx:213]: Got it! Can we add a comment here and/or move this to a function called `getLabel` with this logic? So anybody that comes here can know the reasoning behind this 😃

**Pull request merged**